models/izhikevich.h

```diff
@@ -48,9 +48,7 @@ Implementation of the simple spiking neuron model introduced by Izhikevich
 [1]_. The dynamics are given by:
 
 .. math::
-
-   &dV_m/dt = 0.04 V_m^2 + 5 V_m + 140 - u + I \\
-   &du/dt = a (b V_m - u) \\
```

Comment:
thanks for the changes @PrabhuUdurg 
It looks like this line got delete, can you re-add it, changing u to U_m?

Reply:
Sure) I was just thinking you decided to delete it.